xinference/client.py

```diff
@@ -208,15 +208,15 @@ def streaming_response_iterator(
 
 # Duplicate code due to type hint issues
 def chat_streaming_response_iterator(
-    response_lines: Iterator[bytes],
```

Comment:
Does `streaming_response_iterator` need to be updated?

Reply:
Yes. The respose stream is iterated by chunk instead of by line.

Reply:
I mean the method `streaming_response_iterator`.  You've modified `chat_streaming_response_iterator`, but 
 not `streaming_response_iterator`.

Reply:
> I mean the method `streaming_response_iterator`. You've modified `chat_streaming_response_iterator`, but not `streaming_response_iterator`.

Fixed, thanks. But why two same methods?